resources/conf.edn

```diff
@@ -14,7 +14,7 @@
              :pubkey #profile {:development "auth_pubkey.pem"
                                :test "test_pubkey.pem"
                                :production "/root/prod_pubkey.pem"}
-             :secrets #include "secrets.edn"
+             :secrets #include #join [#env HOME "/.secrets.edn"]
```

Comment:
This bit assumes the secrets file is in the home directory - and the download-secrets.edn file places it there when starting the docker instance.